### DISCUSSION TITLE
链接收藏

### DISCUSSION THREAD

**Discussion by pipi32167** [he3-app/he3-feedback#76] [Ideas]
有时候有些工具he3还没提供，但是又是开发相关的，想集中到一起使用，希望能够提供一个收藏网址添加标题和描述进行搜索的功能。

**Comment by hongchaodeng** [comment]
这个不能用 Chrome bookmarks 么...

**Reply by pipi32167** [reply]
工具集中起来使用...避免分散注意力...

**Reply by hongchaodeng** [reply]
好的，鉴于我们本身也是一个浏览器，这个也是一个很好的 idea!
